usagov/usagov-2021
# Usagov 1514 benefits search theming

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1628

## Description


- Added SVG icons for life event cards
- Updated benefits-search.css to uswds standards
- Updated templates to use fields for life events and benefits search

## Type of Changes

  - Add screenshot showing what it should look like
  - Note that this will look slightly different from the designs because the margin for the design is not the same as the actual site.
![image](https://github.com/usagov/usagov-2021/assets/170120499/cd039c49-062a-41c5-b845-b6f224cc8427)


## Testing Instructions


If there is life events content tagged already, you should just be able to view the benefits search page.

### Requires New Config

### Requires New Content

### Validation Steps
- Are life events showing with the appropriate icons.
- Benefits category show when checked
- Clear selections button works 
- Results show correctly
- Resizing to mobile works 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
